app/controllers/list_controller.rb

```diff
@@ -0,0 +1,46 @@
+class ListController < ApplicationController
+  def index
+    @lists = List.all
+  end
+
+  def edit
+    @list = List.find(params[:id])
+  end
+
+  def new
+    @list = List.new
+  end
+
+  def show
+    @list = List.find(params[:id])
+    @requests = @list.gift_requests
+  end
+
+  def create
+    @list = List.new(list_params.merge(therapist: current_user.role))
+    if @list.save
+      redirect_to action: "show", id: @list.id
+    else
+      flash[:error] = "A problem occurred while creating the wishlist. Please try again."
+      render :new
+    end
+  end
+
+
+  def update
+    @list = List.find(params[:id])
+    render nothing: true, status: @list.update_attributes(list_params) ? :ok : :bad_request
```

Comment:
Needs to redirect to the show page for the updated list (or back to the edit page if there are errors).